launcher/minecraft/launch/LauncherPartLaunch.cpp

```diff
@@ -159,6 +160,74 @@ void LauncherPartLaunch::executeTask()
 
     qDebug() << args.join(' ');
 
+    args.prepend(FS::ResolveExecutable(instance->settings()->get("JavaPath").toString()));
+
+#ifdef Q_OS_LINUX
+    // TODO: Sandboxing setting
+    if (true) {
+        QString actualRuntimeDir = QStandardPaths::writableLocation(QStandardPaths::RuntimeLocation);
+        QString sandboxedRuntimeDir = "/tmp";
+
+        QStringList bwrapArgs{};
+        bwrapArgs << QStandardPaths::findExecutable(BuildConfig.LINUX_BWRAP_BINARY);
+        bwrapArgs << "--unshare-user" << "--unshare-ipc" << "--unshare-pid" << "--unshare-uts" << "--unshare-cgroup";
+        bwrapArgs << "--share-net";
```

Comment:
Is it better to make this allow-list-style like `--unshare-all --unshare-net`? This will also skip users and cgroups isolation when unsupported.